packages/valory/skills/registration_abci/rounds.py

```diff
@@ -44,6 +44,19 @@ class Event(Enum):
     FAST_FORWARD = "fast_forward"
 
 
+class SynchronizedData(BaseSynchronizedData):
```

Comment:
I added this here as we often use `safe_contract_address` in setup and hence need to insist it being propagated through periods